lib/workers/pr/index.ts

```diff
@@ -125,6 +125,12 @@ export function getPlatformPrOptions(
       config.automergeType === 'pr' &&
       config.gitLabAutomerge,
   };
+
+  if (config.gitLabIgnoreApprovals) {
```

Comment:
Why does this need to be in its own "if"?

Reply:
I thought this behavior should be optional, as described in #6577

Reply:
But isn't it optional because it has default value `false`? I think I'm missing something

Reply:
Ah, no, it's me. Confused with another `if` branch. Tried to avoid `gitLabIgnoreApprovals: false` in test snapshots. Will simplify and update them now.